src/Rector/Dynamic/MethodNameReplacerRector.php

```diff
@@ -134,7 +134,11 @@ public function refactor(Node $node): ?Node
             return $this->resolveClassRename($node, $oldToNewMethods, $methodName);
         }
 
-        $this->methodNameChanger->renameNode($node, $oldToNewMethods[$methodName]);
+        if ($node instanceof MethodCall) {
```

Comment:
@TomasVotruba I know we should not check for instances, but I could refactor better this method to fix the [errors](https://travis-ci.org/rectorphp/rector/builds/321886423).